cli/command/cli.go

```diff
@@ -171,6 +170,7 @@ func (cli *DockerCli) Initialize(opts *cliflags.ClientOptions) error {
 
 			opts.Common.TLSOptions.Passphrase = passwd
 			cli.client, err = NewAPIClientFromFlags(opts.Common, cli.configFile)
+			cli.client.NegotiateAPIVersion()
```

Comment:
This doesn't seem like the right place for this. We also don't want to call ping again. I think this line needs to be removed.